client/layout/index.jsx

```diff
@@ -121,5 +123,11 @@ module.exports = React.createClass( {
 					isActive={ translator.isActivated() }/>
 			</div>
 		);
+
+		return (
+			<Provider store={ this.props.store }>
```

Comment:
https://github.com/Automattic/wp-calypso/pull/1496/files#diff-40142ad60b0ab946a566d1d4933afab8L306
This is an ecample, it caught me by surprise too :)